src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2635,21 +2635,24 @@ because xbuild doesn't support framework reference assemblies.
 
   <DetermineJavaLibrariesToCompile
     MonoPlatformJarPath="$(MonoPlatformJarPath)"
+    AndroidSdkDirectory="$(_AndroidSdkDirectory)"
     JavaSourceFiles="@(AndroidJavaSource)" 
     JavaLibraries="@(AndroidJavaLibrary)"
     ExternalJavaLibraries="@(AndroidExternalJavaLibrary)"
     LibraryProjectJars="@(ExtractedJarImports)"
     DoNotPackageJavaLibraries="@(_ResolvedDoNotPackageAttributes)"
     EnableInstantRun="$(_InstantRunEnabled)"
     AdditionalJavaLibraryReferences="@(_AdditionalJavaLibraryReferences)"
-    >
```

Comment:
It would make for a smaller diff if this line was preserved, and the added `AndroidApiLevel` attribute were set alongside the added `AndroidSdkDirectory` attribute.